frontend/src/app/components/LandingPage/HeadingTexts.tsx

```diff
@@ -0,0 +1,20 @@
+import { DM_Serif_Display } from "next/font/google";
+
+const dm_Serif_Display = DM_Serif_Display({
+  weight:'400',
+  style:['italic'],
+  subsets:['latin']
+});
+
+export default function HeadingTexts() {
```

Comment:
Encapsulating the homepage text like this may not be necessary, since we're most likely not re-using this for anything else. It could be safe to probably keep in the homepage to make things less complicated